scipy/stats/morestats.py

```diff
@@ -959,11 +959,26 @@ def boxcox(x, lmbda=None, alpha=None, optimizer=None):
         If ``alpha`` is not None, return the ``100 * (1-alpha)%`` confidence
         interval for `lmbda` as the third output argument.
         Must be between 0.0 and 1.0.
-    optimizer : callable, optional
-        Callable which follows the `optimize.minimize_scalar` interface, with two input
-        arguments (the function over which to optimize and arguments to that
-        function) and `OptimizeResult` as the output object. See
-        `scipy.optimize.minimize_scalar` for more details.
+    optimizer : callable, optional (default: `scipy.optimize.brent`)
```

Comment:
```suggestion
    optimizer : callable, optional
```
Oops. That's not quite true. Rather than being very precise and making this even longer, let's just drop it.